test/EasyRdf/Serialiser/NtriplesTest.php

```diff
@@ -304,4 +304,35 @@ public function testIssue219Unicode()
             $this->assertEquals($expected, $actual);
         }
     }
+
+    /**
+     * Tests combinations of control characters and multibyte characters.
+     */
+    public function testMixedWithControlCharacters()
+    {
+        $serializer = new Ntriples();
+        // Include the NULL byte, a character, a control character,
+        // a multibyte character, a character, and a character outside the BMP.
+        $string = chr(0) . 'a' . chr(31) . '位' . chr(127) . '𐀐';
+
+        $literal = new Literal($string);
+        $actual = $serializer->serialiseValue($literal);
+
+        $this->assertEquals('"\u0000a\u001F\u4F4D\u007F\U00010010"', $actual);
+    }
+
+    /**
+     * Tests that random sequences are not confused with multibyte characters.
+     */
+    public function testUintendedMultibyteCharacter()
```

Comment:
About the part `Uintended`. Do you mean `Unintended`?

Reply:
ouch, yes, let me fix that. By the way, yesterday we tested the changes also in our code, which we have a long list of automated tests, most of them using the serializer multiple times, and the tests were green. Now, our project is not using any weird characters in the tests but still one extra proof for the current PR.

Reply:
Fixed.